README.md

```diff
@@ -33,29 +34,36 @@ WebVR Volume Rendering.
 
 #### Introduction
 
-A web based volume visualizer application to support scientific 3D data built on top of webgl that will facilitate the access from any web browser, plus the benefits of VR.
-
-It was implemented using A-frame, a javascript framework easy to learn, use and deploy. It sits on top of Three.js, which is the most used library to support 3D rendering on the web, and html which is the common tool to write static web pages. A-frame follows a composed based pattern, common on game and graphics engines. Additionally, it uses node js to easy deploy the modules the application is based on.
+A web based 3D volume visualizer application to support scientific 3D data built on top of webgl that will facilitate the access from any web browser, plus the benefits of VR.
+This specific release showcases data from the Narragansett Bay in Rhode Island on different seasons and ocean conditions.
 
 ![Application_Arch](./imgs/archit.png)
 
 <a name="Demo"/>
 
 #### Live Demo
 
-[Click here for live demo](https://datasci.brown.edu/volume-vr/)
+[Click here for live demo](https://bay-viewer.riddc.brown.edu/)
+
+<a name="UIControls"/>
+
+#### UI Controls
```

Comment:
delete this section? it's already in the app/demo

Reply:
I deleted this section